packages/client/tests/functional/0-legacy-ports/batch-find-unique/__snapshots__/tests.ts.snap

```diff
@@ -8,6 +8,10 @@ exports[`0-legacy-ports.batch-find-unique (provider=mysql) findUnique batching 1
 
 exports[`0-legacy-ports.batch-find-unique (provider=postgresql) findUnique batching 1`] = `SELECT "public"."User"."id", "public"."User"."email", "public"."User"."age", "public"."User"."name" FROM "public"."User" WHERE "public"."User"."email" IN ($1,$2,$3,$4) OFFSET $5`;
 
+exports[`0-legacy-ports.batch-find-unique (provider=postgresql, js_neon) findUnique batching 1`] = `SELECT "public"."User"."id", "public"."User"."email", "public"."User"."age", "public"."User"."name" FROM "public"."User" WHERE "public"."User"."email" IN ($1,$2,$3,$4) OFFSET $5`;
+
+exports[`0-legacy-ports.batch-find-unique (provider=postgresql, js_pg) findUnique batching 1`] = `SELECT "public"."User"."id", "public"."User"."email", "public"."User"."age", "public"."User"."name" FROM "public"."User" WHERE "public"."User"."email" IN ($1,$2,$3,$4) OFFSET $5`;
```

Comment:
Yes! I think we can. This has an implication because of how jest works: if your snapshots have the same name as the regular postgres ones, then you can only run either one flavor at a time, not all at once. And also means for snapshots that don't match 100%, we'll need to have custom snapshot names again. Ideally things just match and work strictly the same.